src/version/actions/contrastModeAction.js

```diff
@@ -4,6 +4,4 @@ export const CONTRAST_MODE_ACTION = createRequestTypes('CONTRAST_MODE_ACTION');
 
 export const contrastModeAction = {
     request: data => ({ type: CONTRAST_MODE_ACTION.REQUEST, data }),
```

Comment:
IMO, you don't need to export an object since there is only one action. Something like 

```js
export const setContrastMode = data => ({ type: CONTRAST_MODE_ACTION.REQUEST, data })
``` 

Will be enough:

Also, I'm not fan of the generic `data` argument here. It should be clear by reading the function signature what data it expects. By reading at the reducer, I understand that `data` is treated as a boolean, so I'd write the actino as follows:

```js
export const setContrastMode = isEnabled => ({ type: CONTRAST_MODE_ACTION.REQUEST, isEnabled })
```